pace-util/pace/util/zarr_monitor.py

```diff
@@ -119,9 +119,10 @@ def store(self, state: dict) -> None:
         """Append the model state dictionary to the zarr store.
 
         Requires the state contain the same quantities with the same metadata as the
-        first time this is called. Quantities are stored with dimensions [time, rank]
-        followed by the dimensions included in any one state snapshot. The one exception
-        is "time" which is stored with dimensions [time].
+        first time this is called. Dimension order metadata may change between calls
+        so long as the set of dimsensions is the same. Quantities are stored with
+        dimensions [time, rank] followed by the dimensions included in any one state
+        snapshot. The one exception is "time" which is stored with dimensions [time].
```

Comment:
Have to make this distinction since your edit.
```suggestion
        dimensions [time, rank] followed by the dimensions included in the first state
        snapshot. The one exception is "time" which is stored with dimensions [time].
```